providers/os/provider/platform.go

```diff
@@ -26,6 +27,7 @@ type PlatformFingerprint struct {
 	Runtime       string
 	Kind          string
 	RelatedAssets []PlatformFingerprint
+	idDetectors   []string
```

Comment:
to avoid confusion, we may want to rename this to `activeIdDetectors`

Reply:
Changed